src/App.scss

```diff
@@ -0,0 +1,62 @@
+@import './styles/variables';
+
+@font-face {
+  font-family: 'Montserrat';
+  src: url('../public/assets/Montserrat.ttf') format('truetype');
+}
+
+body {
+  color: $primary-color;
+  background-color: $background-color;
+
+}
+
+* {
+  box-sizing: border-box;
+  font-family: 'Montserrat';
+}
+
+.app-container {
+  padding: 24px;
+}
+
+.movie-tile-container {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(250px, 1fr));
+  justify-content: space-around;
+  align-items: center;
+  gap: 20px;
+  margin-top: 1em;
+}
+
+.movie-genre-sort-by-container {
+  display: grid;
+  grid-template-columns: 1fr auto 1fr;
+}
+
+.genreSelectContainer {
+  justify-self: start;
+  margin-top: 1.3em;
+}
+
+.sortByContainer {
+  justify-self: end;
+  margin-top: 0.3em;
+}
+
+.bold-br {
+  display: block;
+  width: 100%;
+  height: 2px;
+  background-color: $dark-label-color;
+  box-shadow: 1px 3px 2px #000000;
+  margin: 1px 0;
+}
+
+p {
+  margin: 0;
+}
+
+.movie-count {
+  margin-top: 1em;
+}
```

Comment:
Please nest the styling. Parent -> { child  -> { next child } } }

Reply:
Nested styling.